extensions/cornerstone/src/Viewport/OHIFCornerstoneViewport.tsx

```diff
@@ -404,17 +409,32 @@ function _jumpToMeasurement(
     displaySetInstanceUID
   );
 
-  const imageIdIndex = referencedDisplaySet.images.findIndex(
-    i => i.SOPInstanceUID === SOPInstanceUID
-  );
-
   // Todo: setCornerstoneMeasurementActive should be handled by the toolGroupManager
   //  to set it properly
   // setCornerstoneMeasurementActive(measurement);
 
   viewportGridService.setActiveViewportIndex(viewportIndex);
 
-  if (getEnabledElement(targetElement)) {
+  const enableElement = getEnabledElement(targetElement);
+  if (enableElement) {
+    // See how the jumpToSlice() of Cornerstone3D deals with imageIdx param.
+    const viewport = enableElement.viewport as IStackViewport | IVolumeViewport;
+    const { imageIds } = viewport;
+    let imageIdIndex;
+
+    for (let i = 0; i < imageIds.length; i++) {
```

Comment:
You should use findIndex again, rather than implementing findIndex manually.

Reply:
Done in 52b4f8b6b14a00b6d85890dd3e7aea1580b1581c